tests/functional/clades/expected-output-default.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
This doesn't match the updated spec in the PR conversation:
```
{
  "nodes": {
    "ARG/Cordoba-12873-61/2020": {
      "clade_membership": "20A"
    }
  },
  "branches": {
    "NODE_0000000": {
      "labels": {
        "clade": "19A"
      }
    }
  }
}
```

Reply:
I glossed over this as I copied & pasted. The PR, as it currently stands, uses the structure above, but the key name in both the nodes and branches is the same (`"clade"` in this case).

Reply:
Yes, sorry that was a typo on my part. I should have written the example like this such that all annotations from the same run of `augur clades` use the same `--attribute-name`:

```json
{
  "nodes": {
    "ARG/Cordoba-12873-61/2020": {
      "clade": "20A"
    }
  },
  "branches": {
    "NODE_0000000": {
      "labels": {
        "clade": "19A"
      }
    }
  }
}
```

Reply:
Got it. This makes much more sense now. Thank you for clarifying.